figma.html

```diff
@@ -0,0 +1,447 @@
+        <span class="carousel-control-prev-icon" aria-hidden="true"></span>
+        <span class="visually-hidden">Previous</span>
+      </button>
+      <button class="carousel-control-next" type="button" data-bs-target="#carouselExampleCaptions" data-bs-slide="next">
+        <span class="carousel-control-next-icon" aria-hidden="true"></span>
+        <span class="visually-hidden">Next</span>
+      </button>
+    </div>
+</section>
+  <section class="p-3">
+    <div class="container">
+        <h2 class="text-center mb-4 ">
+            Check our latest article
+        </h2>
+        <hr class="custom-hr mb-5">
+
+        <div class="row text-center g-4">
+            <div class="col-md">
+                <div class="card" >
+                    <img src="img/igla.png" class="card-img-top" alt="">
+                    <div class="card-body">
+                      <h2 class="fs-5 text-start">Disease detection, check 
+                        up in the laboratory
+
+                      </h2>
+                        <p class="card-text text-start">In this case, the role of the health laboratory is very important to do
+                          a disease detection...</p>
+                        <a href="" class="fw-bold">Learn more -></a>
+                    </div>
+                </div>
+            </div>
+        
+            <div class="col-md">
+                <div class="card" >
+                    <img src="img/doca.png" class="card-img-low"  alt="">
+                    <div class="card-body">
+                      <h2 class="fs-5 text-start">Herbal medicines that are 
+                        safe for consumption
+
+                      </h2>
+                        <p class="card-text text-start" >Herbal medicine is very widely used at this time because of its very good for your health...</p>
+                        <a href="" class="fw-bold">Learn more -></a>
+                    </div>
+                </div>
+            </div>
+        
+            <div class="col-md">
+                <div class="card">
+                    <img src="img/zena.png" class="card-img-top" alt="">
+                    <div class="card-body">
+                      <h2 class="fs-5 text-start">Natural care for healthy 
+                        facial skin
+
+                      </h2>
+                        <p class="card-text text-start">A healthy lifestyle should start from now and also for your skin health.
+                          There are some...</p>
+                          <a href="" class="fw-bold">Learn more -></a>
+                    </div>
+                </div>
+            </div>
+        </div>
+        <div class="d-flex align-items-center justify-content-center">             
+          <a class="btn btn-outline-primary m-5 btn-lg" href="" role="button">View all</a>         
+        </div>
+</section>
+  <footer class="bg-primary text-center text-lg-start bg-body-tertiary text-white p-5 mt-3">

```

Comment:
bg-primary je prejak s obzirom na dizajn, trebalo bi malo smanjiti odnosno možeš bg-primary klasu overridat s bojom s dizajna, a nekad u budućnosti to možemo s varijablama